docs/docs/Support/security.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Security notice
+slug: /security
+---
+
+The Langflow UI is an IDE and code execution platform, which means that Langflow is inherently capable of executing arbitrary, developer-provided code.
+By design, the Langflow UI includes a code editor allowing developers to author and execute arbitrary Python with full access to the host Langflow backend process, filesystem, and network.
+In addition, various components wrap and make use of code execution, in some cases using AI models to generate the code to be executed based on user input.
+
+Given these product features, the Langflow application makes no attempt to prevent malicious activity between different users of a given Langflow process, or to secure the execution environment of the Langflow process, such as preventing access to portions of the local disk. To the extent that limitations are imposed on the access provided to different users, such as limiting flow visibility to a user's flows, these limitations are intended to improve usability, not to impose secure access policies.
+For example, users of Langflow can always access the underlying database connection directly.
+
+The security of code and flows written using Langflow is the developer’s responsibility.
```

Comment:
"The developer" is not clear whether it means LF developers or the reader. I suggest being more specific:

```suggestion
The security of code and flows written using Langflow is your responsibility as a developer.
```